pkg/common/purl/purl.go

```diff
@@ -77,6 +77,10 @@ func PurlTypeToEcosystem(purlType string) (lockfile.Ecosystem, error) {
 		"rubygems":              lockfile.BundlerEcosystem,
 		packageurl.TypeGithub:   models.EcosystemGitHubActions,
 		"actions":               models.EcosystemGitHubActions,
+		"vscode":                models.EcosystemVSCodeExtensions,
```

Comment:
This a technical debt. At some point in future, we need to clean this up and move to using DRY as the source of truth for PURL parsing.

Reply:
@Sahilb315 FYI. Put something in the backlog for this.